packages/host/tests/integration/components/ai-assistant-panel-test.gts

```diff
@@ -2664,6 +2664,25 @@ module('Integration | ai-assistant-panel', function (hooks) {
     );
 
     await waitFor('[data-test-message-idx="0"]');
+    assert
+      .dom('button.code-copy-button')
+      .exists('the copy code to clipboard button exists');
+
+    // assert that new messages don't destabilize the RoomMessage component
```

Comment:
What actually asserts this?

Reply:
Adding a new event would previously destabilize the RoomMessage component (as described in the PR description). The copy button and Monaco editors are from a modifier that adds DOM. A rerender of this component would blow away the DOM added by their modifier.